src/hw_mainline.js

```diff
@@ -289,6 +291,8 @@ var enableAIN = function (callback) {
     return (ainPrefix);
 };
 
+// HACK to work on AI.
+const aimap = [0, 1, 3, 2, 7, 6, 4];  // Fixes AI's AIN mapping.
```

Comment:
This needs to be removed now. I have the mapping as part of isAI.

Reply:
Done.